### PR TITLE
TimezoneConverter: switch HTML to template

### PR DESCRIPTION
**Pull request by cwallen**
Part of https://github.com/duckduckgo/zeroclickinfo-goodies/issues/1163

It drops the styles, but I didn't see a way to keep that without getting more custom than seemed worth it.

Before: 
screenshot 2015-07-19 21 11 43

After:
screenshot 2015-07-19 21 06 20

### CODE REVIEW

**Comment by mintsoft** [comment]
@abeyang is the actual answer clear enough?

**Comment by abeyang** [comment]
Hi @cwallen , this looks way better than before! However, let's put the direct answer first, and the query second. For example, your query should produce:

> **2:00 EST**
> Convert Timezone: 10:00 MSK to EST

-- and "8:00 msk to est" would produce:

> **23:00 EST (one day prior)**
> Convert Timezone: 8:00 MSK to EST

**Comment by cwallen** [comment]
Ok, I can make those changes later tonight.  Would it be better to show the input verbatim as entered, or parsed like in the first part of the current answer?

**Comment by mintsoft** [comment]
@cwallen I'd say parsed; that way if there's an issue it lets the user know how it was interpreted

**Comment by cwallen** [comment]
@abeyang These look good?

screenshot 2015-07-23 08 47 22
screenshot 2015-07-23 08 47 42

**Comment by cwallen** [comment]
Finally got some time to finish this up. Should look like my last screenshots above.  Also did a bit of refactoring to clean up code that was doing the same thing over again for the input and output times.

File: lib/DDG/Goodie/TimezoneConverter.pm [hunk 1]
```diff
@@ -133,63 +133,65 @@ handle query => sub {
...
-    my $input_time  = $hours + $minutes / 60 + $seconds / 3600 + $pm;
-    my $output_time = $input_time + $modifier;
-    for ( $input_time, $output_time ) {
-        my $days = "";
-        if ( $_ < 0 ) {
-            my $s = $_ <= -24 ? 's' : "";
-            $days = sprintf ', %i day%s prior', $_ / -24 + 1, $s;
+    $input->{time}  = $hours + $minutes / 60 + $seconds / 3600 + $pm;
+    $output->{time} = $input->{time} + $modifier;
+    for ( $input, $output ) {
```
Comment (mintsoft) [inline lib/DDG/Goodie/TimezoneConverter.pm]: you could do `for my $io ($input,$output);`

Comment (cwallen) [inline lib/DDG/Goodie/TimezoneConverter.pm]: Fixed.

**Comment by moollaza** [comment]
@cwallen thanks a lot! This LGTM :+1:

**Pull request merged**